tests/cypress/support/commands.js

```diff
@@ -56,4 +61,15 @@ Cypress.Commands.add("checkA11yOfPage", () => {
      */
     true // does not fail tests for ally violations
   );
+
+  // check accessibility using pa11y
+  cy.pa11y({
+    threshold: 10,
```

Comment:
Can set a threshold of number of errors that you're ok with to still have it pass. We can have this be whatever number we want, but I wanted to start a discussion about it first. You can also set limits for notices and warnings. 

See the whole list of [pa11y options here](https://github.com/pa11y/pa11y#configuration).